### PR TITLE
Put all JSON editor code and dependencies in one file so we can install it from Github

### PR DESCRIPTION
**Pull request by azer**

### CODE REVIEW

**Comment by azer** [comment]
@floelhoeffel There was a typo, please try it again

**Comment by floelhoeffel** [comment]
Works - thanks!

File: samples/json-editor/index.html [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+  <body>
+    <script>
+     /*
+      * Custom JSON editor extension
+      * Displays JSON object, validates on the fly and saves if its valid
+      */
+
+           +function() {
+
+               if (typeof window.contentfulExtension === 'undefined') {
```
Comment (jelz) [inline samples/json-editor/index.html]: can be removed

File: samples/json-editor/index.html [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+                   // Backwards compatibility with devloper preview program.
+                   window.contentfulExtension = window.contentfulWidget;
+               }
+               if (typeof window.contentfulExtension === 'undefined') {
```
Comment (jelz) [inline samples/json-editor/index.html]: can be removed

File: samples/json-editor/index.html [hunk 3]
```diff
@@ -0,0 +1,267 @@
+<!DOCTYPE html>
+<html>
+  <head>
```
Comment (jelz) [inline samples/json-editor/index.html]: I'd add `<meta charset="utf-8">`

File: samples/json-editor/index.html [hunk 4]
```diff
@@ -0,0 +1,267 @@
...
+    </style>
+  </head>
+  <body>
+    <script>
+     /*
+      * Custom JSON editor extension
+      * Displays JSON object, validates on the fly and saves if its valid
+      */
+
+           +function() {
```
Comment (jelz) [inline samples/json-editor/index.html]: I'd skip self invoking wrapper

File: samples/json-editor/index.html [hunk 5]
```diff
@@ -0,0 +1,267 @@
...
+     }
+
+     .redo:after {
+         content: 'Redo';
+     }
+
+    </style>
+  </head>
+  <body>
+    <script>
```
Comment (jelz) [inline samples/json-editor/index.html]: can we start code at col 0

File: samples/json-form-editor/index.html [hunk 6]
```diff
@@ -0,0 +1,303 @@
+<!DOCTYPE html>
+<html>
+  <head>
```
Comment (jelz) [inline samples/json-form-editor/index.html]: charset

File: samples/json-form-editor/index.html [hunk 7]
```diff
@@ -0,0 +1,303 @@
...
+             };
+         }
+
+     }(JSONEditor));
+
+
+
+     // https://github.com/jdorn/json-editor/blob/master/examples/advanced.html
+     // TODO: Make this configurable on Data Type level.
+     /*eslint quotes: [2, "double"]*/
```
Comment (jelz) [inline samples/json-form-editor/index.html]: remove, no eslint setup here

**Pull request merged**